digits/extensions/view/interface.py

```diff
@@ -49,6 +49,32 @@ def get_header_template(self):
         """
         return None, None
 
+    def get_ng_templates(self):
```

Comment:
Decided to rename ng_header and ng_footer to app_begin and app_end. It turns out that trying to work these into the header_html would make it too convoluted.